src/controller/controldevice/controller/mapping/ControllerMapping.h

```diff
@@ -7,7 +7,8 @@ namespace Ship {
 
 #define MAPPING_TYPE_GAMEPAD 0
 #define MAPPING_TYPE_KEYBOARD 1
-#define MAPPING_TYPE_UNKNOWN 2
+#define MAPPING_TYPE_MOUSE 2
+#define MAPPING_TYPE_UNKNOWN 3
```

Comment:
Unknown should probably be hard coded to something like -1

Reply:
that's was definitely a me thing, i don't remember if we hit snags with that in the initial implementation or not

Reply:
Seems like it wouldnt cause any issues now on 2ship and soh, so Im changing it